src/Command/RunCommand.php

```diff
@@ -41,6 +43,17 @@ protected function configure(): void
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
+        $this->trackingService->track('php_version', [
+            'php_version' => \PHP_MAJOR_VERSION . '.' . \PHP_MINOR_VERSION,
+        ]);
+
+        try {
```

Comment:
wouldn't it be better to catch the exception in the state class? and give something back like "could not detect mysql version because server is gone"